scripts/release

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const execSync = require('child_process').execSync;
+const minimist = require('minimist');
+const standardVersion = require('standard-version');
+const chalk = require('chalk');
+const args = minimist(process.argv.slice(2));
+
+const DEFAULT_UPSTREAM_REPO_ORIGIN = 'origin';
+const DEFAULT_UPSTREAM_REPO_BRANCH = 'master';
+const origin = args['repo-origin'] || DEFAULT_UPSTREAM_REPO_ORIGIN;
+const branch = args['repo-branch'] || DEFAULT_UPSTREAM_REPO_BRANCH;
+
+standardVersion({
+  dryRun: args['dry-run'],
+  prerelease: args.prerelease,
+  sign: args.sign,
+  releaseAs: args['release-as']
+}).then(() => {
+  if (!args['dry-run']) {
+    console.log(chalk.blue('ℹ'), `Pushing release commit to origin '${origin}' on branch '${branch}'...`);
+    try {
+      const output = execSync(`git push ${origin} ${branch} --follow-tags`).toString();
+      console.log(chalk.green(output));
+      console.log(chalk.green('✔'), 'Successfully pushed release commit');
+    } catch (e) {
+      console.log(chalk.red('✘'), 'Couldn\'t push release commit. Please check the error above.');
+      return { error: true };
+    }
+  } else {
+    console.log(chalk.blue('ℹ'), 'This is a dry run. Nothing\'s being pushed.');
+  }
+}).then(error => {
+  if (error) {
+    return error;
+  }
+  if (!args['dry-run']) {
+    console.log(chalk.blue('ℹ'), 'Publishing new Embark version on npm...');
+    try {
+      const output = execSync('npm publish').toString();
```

Comment:
This command needs to be extended with `--tag prerelease` in case `args['prerelease']` is set.